src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="/" 
+      method="post" 
+      class="search-form" 
+      data-qa="big"
+    >
+      <label for="search-input" class="search-icon"></label>
+
+      <input 
+        type="search" 
+        id="search-input"
+        name="search"
+        class="input"
+        data-qa="keypress"
+        placeholder="Try &#34;Los Angeles&#34;"
+      >
```

Comment:
```suggestion
      <label for="search-input" class="search-icon">
        <input 
          type="search" 
          id="search-input"
          name="search"
          class="input"
          data-qa="keypress"
          placeholder="Try &#34;Los Angeles&#34;"
        >
      </label>
```
input should be wrapped with the label